templates/default/stripe_preferences.html.twig

```diff
@@ -43,8 +43,8 @@
                     <code>{{ webhook_url }}</code>
                 </div>
                 <div class="inline field">
-                    <label>{{ _T("Stripe Webhook events:", "stripe") }}</label>
-                    <code>payment_intent.succeeded</code> {{ _T("and") }} <code>invoice.payment_succeeded</code>
+                    <label>{{ _T("Stripe Webhook event:", "stripe") }}</label>
+                    <code>payment_intent.succeeded</code>
```

Comment:
So. The reference to the `invoice.payment_succeeded` event on the settings page is the only thing that should actually be dropped as it is only intended to be used by a custom form outside of Galette. 

The plugin itself doesn't use this event, and if someone adds it and uses the invoices feature from Stripe's dashboard, this will fill up the plugin's logs with empty duplicated entries.

I'll open another PR for that.

Shall we keep this PR opened as a draft and wait that @manuelh78 tests the develop branch and gives us feedback ?